filebeat/processor/add_kubernetes_metadata/matchers_test.go

```diff
@@ -36,20 +37,32 @@ func TestLogsPathMatcher_InvalidSource3(t *testing.T) {
 func TestLogsPathMatcher_VarLibDockerContainers(t *testing.T) {
 	cfgLogsPath := "" // use the default matcher configuration
 	source := fmt.Sprintf("/var/lib/docker/containers/%s/%s-json.log", cid, cid)
+	if runtime.GOOS == "windows" {
+		cfgLogsPath = "D:\\containers\\logs"
+		source = fmt.Sprintf("D:\\containers\\logs\\%s\\%s-json.log", cid, cid)
+	}
 	expectedResult := cid
 	executeTest(t, cfgLogsPath, source, expectedResult)
 }
 
 func TestLogsPathMatcher_VarLogContainers(t *testing.T) {
 	cfgLogsPath := "/var/log/containers/"
 	source := fmt.Sprintf("/var/log/containers/kube-proxy-4d7nt_kube-system_kube-proxy-%s.log", cid)
+	if runtime.GOOS == "windows" {
+		cfgLogsPath = "D:\\containers\\logs"
+		source = fmt.Sprintf("D:\\containers\\logs\\%s\\%s-json.log", cid, cid)
+	}
 	expectedResult := cid
 	executeTest(t, cfgLogsPath, source, expectedResult)
 }
 
 func TestLogsPathMatcher_AnotherLogDir(t *testing.T) {
 	cfgLogsPath := "/var/log/other/"
 	source := fmt.Sprintf("/var/log/other/%s.log", cid)
+	if runtime.GOOS == "windows" {
+		cfgLogsPath = "D:\\containers\\logs"
```

Comment:
Here the test is testing "another" log directory, so this should be different to the previous one.